docs/Get-PSReadLineOption.md

```diff
@@ -1,47 +1,109 @@
 ---
+external help file: Microsoft.PowerShell.PSReadLine.dll-Help.xml
+keywords: powershell,cmdlet
+locale: en-us
+Module Name: PSReadLine
+ms.date: 12/07/2018
+online version: http://go.microsoft.com/fwlink/?LinkId=821450
 schema: 2.0.0
-external help file: Microsoft.PowerShell.PSReadLine2.dll-help.xml
+title: Get-PSReadLineOption
 ---
 
 # Get-PSReadLineOption
 
 ## SYNOPSIS
-
 Returns the values for the options that can be configured.
 
 ## SYNTAX
 
 ```
-Get-PSReadLineOption
+Get-PSReadLineOption [<CommonParameters>]
 ```
 
 ## DESCRIPTION
 
-Get-PSReadLineOption returns the current state of the settings that can be configured by Set-PSReadLineOption.
+The `Get-PSReadLineOption` cmdlet returns the current state of the settings that can be configured
+by using the `Set-PSReadLineOption` cmdlet. You can use the returned object to change
+**PSReadLine** options. This provides a slightly simpler way to set syntax coloring options for
+multiple kinds of tokens.
 
-The object returned can be used to change PSReadLine options.
+## EXAMPLES
 
-This provides a slightly simpler way of setting syntax coloring options for multiple kinds of tokens.
+### Example 1: Get options and their values
 
-## EXAMPLES
+```powershell
```

Comment:
If this example is included, there should be a note that the example output is dependent on which version of PowerShell you use. The *Color options look different if you run this in Windows PowerShell.

Reply:
The purpose of the PR was to update the documentation for PSReadLine v2. If you install PSReadLine v2 on Windows PowerShell 5.1 does it work differently?

Reply:
The output will look different, yes, you'll see the following in Windows PowerShell:

```CommandColor                           : "$([char]0x1b)[93m"
CommentColor                           : "$([char]0x1b)[32m"
ContinuationPromptColor                : "$([char]0x1b)[37m"
DefaultTokenColor                      : "$([char]0x1b)[37m"
EmphasisColor                          : "$([char]0x1b)[96m"
ErrorColor                             : "$([char]0x1b)[41;93m"
KeywordColor                           : "$([char]0x1b)[92m"
MemberColor                            : "$([char]0x1b)[97m"
NumberColor                            : "$([char]0x1b)[97m"
OperatorColor                          : "$([char]0x1b)[90m"
ParameterColor                         : "$([char]0x1b)[90m"
SelectionColor                         : "$([char]0x1b)[30;47m"
StringColor                            : "$([char]0x1b)[36m"
TypeColor                              : "$([char]0x1b)[37m"
VariableColor                          : "$([char]0x1b)[92m"
```

Note the `$([char]0x1b)` instead of `` `e ``.